docs/tempo/website/_index.md

```diff
@@ -8,6 +8,19 @@ aliases:
 
 <p align="center"><img src="logo_and_name.png" alt="Tempo Logo"></p>
 
-Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki.
+Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki. Tempo can be used with any of the open-source tracing protocols, including Jaeger, Zipkin, and OpenTelemetry.
+
+Tempo has strong integrations with a number of existing open source tools including 
+
+- Grafana! Grafana ships with native support for Tempo using the built-in [Tempo data source](https://grafana.com/docs/grafana/latest/datasources/tempo/).
+- Grafana Loki - Loki, with its powerful query language [LogQL v2](https://grafana.com/blog/2020/10/28/loki-2.0-released-transform-logs-as-youre-querying-them-and-set-up-alerts-within-loki/) allows us to filter down on requests that we care about, and jump to traces using the [Derived fields support in Grafana](https://grafana.com/docs/grafana/latest/datasources/loki/#derived-fields).
+- Prometheus exemplars - which allow to jump from Prometheus metrics to Tempo traces by clicking on recorded exemplars. Read more about this integration in this [blog post](https://grafana.com/blog/2021/03/31/intro-to-exemplars-which-enable-grafana-tempos-distributed-tracing-at-massive-scale/).
+
+<p align="center"><img src="getting-started/assets/trace_custom_metrics_dash.png" alt="Trace visualization in Grafana "></p>
+
+Grafana Tempo builds an index on the high cardinality trace-id field and uses an object store as backend which allows for high parallelization of queries.
```

Comment:
This reads ambiguous to me. What design feature makes it possible to have highly parallel queries, the index, the object store, or both?

Style-wise, should we hyphenate "high-cardinality" when used as an adjective?

Reply:
Yes, to high-cardinality as an adjective. Good catch. 

I asked the Tempo team about what enabled the highly parallel queries. Tempo is designed around a log-structure merge tree. Having blocks in object storage means it can query many blocks at once (thanks @mdisibio !).